lib/Browser.js

```diff
@@ -250,6 +250,18 @@ class BrowserContext extends EventEmitter {
     return this._browser.targets().filter(target => target.browserContext() === this);
   }
 
+  /**
+   * @return {!Promise<!Array<!Puppeteer.Page>>}
+   */
+  async pages() {
+    const pages = await Promise.all(
```

Comment:
Let's use this method to build browser.pages().

Reply:
Done.